src/components/NavBar.vue

```diff
@@ -18,6 +18,7 @@
         />
         <span class="hidden-sm-and-down">Ambianic</span>
       </v-toolbar-title>
+      <v-spacer />
```

Comment:
@vickywane v-spacer appears to be an adequate fix to the issue of navbar icon alignment. However I now observe an overlap of the timeline pane over the navbar on bigger screens. See screenshot. This timeline issue is not present on small screens.

<img width="578" alt="Screen Shot 2021-07-21 at 5 33 16 PM" src="https://user-images.githubusercontent.com/2234901/126507256-00f21aed-bac3-46ae-acda-d94c7c1c8ece.png">

<img width="355" alt="Screen Shot 2021-07-21 at 5 37 40 PM" src="https://user-images.githubusercontent.com/2234901/126507321-0fd49cef-c234-4c5c-bd8f-635217f31940.png">

Reply:
BTW, this timeline pane overlap issue is not present on big screens nor small screens with the current production deployment. See screenshots below.

<img width="850" alt="Screen Shot 2021-07-21 at 5 39 26 PM" src="https://user-images.githubusercontent.com/2234901/126507677-83764f30-a727-459a-b7a7-4eaf57df8e42.png">

<img width="245" alt="Screen Shot 2021-07-21 at 5 40 16 PM" src="https://user-images.githubusercontent.com/2234901/126507797-e43aae70-62fb-497d-853f-8e9809d51ac3.png">